virttest/utils_misc.py

```diff
@@ -2281,7 +2281,7 @@ def _get_backend_dir(params):
     Get the appropriate backend directory. Example: backends/qemu.
     """
     return os.path.join(data_dir.get_root_dir(), 'backends',
-                        params.get("vm_type"))
+                        params.get("vm_type", ""))
```

Comment:
I believe the docstring is correct. You always get the backend directory and if you don't provide the vm_type you get the main backends dir, which could work for no-type tests... (it works)